demo/list-box-presentation-demos.html

```diff
@@ -27,7 +27,7 @@ <h3>Separators</h3>
     <h3>Customizing listbox item label</h3>
     <vaadin-demo-snippet id="list-box-presentation-demos-custom-label">
       <template preserve-content>
-        <label>User</label>
+        <b>User</b>
```

Comment:
Let's resolve the open discussion about this first without changing it back and forth.

Reply:
See above https://github.com/vaadin/vaadin-list-box/pull/63#pullrequestreview-311572853